op-acceptance-tests/tests/interop/proofs/challenger_test.go

```diff
@@ -0,0 +1,39 @@
+package proofs
+
+import (
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-devstack/dsl"
+	"github.com/ethereum-optimism/optimism/op-devstack/dsl/proofs"
+	"github.com/ethereum-optimism/optimism/op-devstack/presets"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+)
+
+func TestChallengerPlaysGame(gt *testing.T) {
+	// Setup
+	t := devtest.SerialT(gt)
+	sys := presets.NewSimpleInterop(t)
+	sys.L1Network.WaitForOnline()
+
+	badClaim := common.HexToHash("0xdeadbeef00000000000000000000000000000000000000000000000000000000")
+	attacker := fundAttackerWallet(t, sys, eth.OneEther.Mul(2))
```

Comment:
```suggestion
	attacker := sys.FunderL1.NewFundedEOA(eth.Ether(2))
```
Verifying the funds were allocated should be handled by the DSL so the tests can stay simple.